### DISCUSSION TITLE
Privacy concerns over osmand according to fdroid

### DISCUSSION THREAD

**Discussion by Raj2032** [osmandapp/OsmAnd#15437] [Q&A] [answered]
Hey guys I have some privacy concerns over this:

According to fdroid's anti-features for this app:

https://f-droid.org/en/packages/net.osmand.plus/

![image](https://user-images.githubusercontent.com/29293830/194258128-e08c0cb4-d7d5-4490-9f69-1e3e0506bc71.png)

I thought it was a privacy oriented app, or is it cause it uses google play services (that can be switched over) that is why fdroid shows antifeatures for this?

Is this the official osmand uploaded by the osmand team?

**Comment by scaidermern** [comment, marked as answer]
> Is this the official osmand uploaded by the osmand team?

No. The F-Droid version is not maintained by the OsmAnd developers. These are inofficial releases.

**Reply by Raj2032** [reply]
I see thanks

**Reply by johnea** [reply]
Just for completeness, does this mean that the official apk of osmand on the play store does NOT export user data? As "telemetry" or any other means?

Of course, when someone intentionally saves to the internet, they are exporting their data, but this is a user action. I'm asking about osmand retrieving user data for it's own purposes...

**Comment by licaon-kter** [comment]
> The F-Droid version is not maintained by the OsmAnd developers. These are inofficial releases.

Love how the answer makes one think that these are issues ONLY in the "inofficial" version somehow introduced by F-Droid. <3
